src/XRP/default-xrp-client.ts

```diff
@@ -426,6 +428,27 @@ export default class DefaultXrpClient implements XrpClientDecorator {
     return XrpTransaction.from(getTransactionResponse, this.network)
   }
 
+  /**
+   * Enable Deposit Authorization for this XRPL account.
+   * @see https://xrpl.org/depositauth.html
```

Comment:
nit: 1 line break after summary

Reply:
I basically follow this guide: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/